test/Microsoft.DotNet.SourceBuild.Tests/assets/SdkContentTests/ApiDiff.suppression

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
```

Comment:
Have you analyzed these diffs to know if they are expected or if they can/should be addressed?  If not I think that can be handled via a follow-up issue.

Reply:
Some of these are expected, but we definitely need to investigate issues in few assemblies.